src/manage_sql.c

```diff
@@ -36920,15 +36967,22 @@ create_config_from_scanner (const char *scanner_id, const char *name,
     }
   quoted_name = sql_quote (name ?: "");
   quoted_comment = sql_quote (comment ?: "");
+  if (usage_type && strcasecmp (usage_type, "policy") == 0)
+    quoted_usage_type = g_strdup ("policy");
```

Comment:
Not really quoted ;)

Reply:
I've renamed it to "actual_usage_type" in 1bc4ce35b6e18ab855179403400f92d0f607ccaa and also changed it to a const char*, avoiding the allocation and freeing as it's using fixed values.